sanic/__main__.py

```diff
@@ -87,15 +88,53 @@ def main():
     parser.add_bool_arguments(
         "--access-logs", dest="access_log", help="display access logs"
     )
-    parser.add_argument(
+    group_worker = parser.add_mutually_exclusive_group()
+    group_worker.add_argument(
         "-w",
         "--workers",
         dest="workers",
         type=int,
         default=1,
-        help="number of worker processes [default 1]\n ",
+        help="Number of worker processes [default 1]",
+    )
+    group_worker.add_argument(
+        "--fast",
+        action="store_true",
+        help=(
+            "Run with maximum allowable processes.\nThis should be equal "
+            "to the number of CPU processes available.\n "
```

Comment:
This message is hard to understand. Maybe something along the lines of `use all available CPUs`?

Reply:
It is worth a note that this does not (until 22.3) disable access logs, or change logging level (not sure if the latter would affect benchmarks). I suppose this is fine if at 22.3 the default mode is steered towards or even merged with the production mode.

Reply:
I threw together a quick little test to check the performance of a simple loop with different logging.

```python
import logging

import timy

LOOPS = 10_000
RANGE = 1_000
LOG = logging.getLogger(__name__)


@timy.timer(loops=LOOPS)
def normal():
    LOG.setLevel(logging.WARNING)
    c = 0
    for i in range(RANGE):
        c += 1


@timy.timer(loops=LOOPS)
def with_logging_warning():
    LOG.setLevel(logging.WARNING)
    c = 0
    for i in range(RANGE):
        c += 1
        LOG.debug("something")


@timy.timer(loops=LOOPS)
def with_logging_debug():
    LOG.setLevel(logging.DEBUG)
    c = 0
    for i in range(RANGE):
        c += 1
        LOG.debug("something")


normal()
with_logging_warning()
with_logging_debug()
```

The results on my machine:

```
Timy executed (normal) for 10000 times in 0.338087 seconds
Timy best time was 0.000031 seconds
Timy executed (with_logging_warning) for 10000 times in 2.469493 seconds
Timy best time was 0.000234 seconds
Timy executed (with_logging_debug) for 10000 times in 77.279574 seconds
Timy best time was 0.007269 seconds
```

Conclusion: changing the logging level has a huge impact on performance. But, even the existence of the not impacted log call is not trivial either. If we are steering towards trying to impress benchmarks, then limiting or removing unused `log` calls completely should make a difference.